Gemfile

```diff
@@ -9,7 +9,7 @@ gem "pg"
 gem "autoprefixer-rails"
 gem "coffee-rails"
 gem "foundation-icons-sass-rails"
-gem "foundation-rails"
+gem "foundation-rails", "~> 6.2.1"
```

Comment:
Why do we need to specify sperm operator here with exact version? I guess updated `Gemfile.lock` should fix version.